qcore.c

```diff
@@ -121,13 +121,110 @@ static  Request request;
 
 
 void the_exception(void) {
+    //TODO: Is compiler error about invalid register name valid?
 	register int otherpidRegister asm("r22");	//read other pid for unblock

```

Comment:
In case anybody has seen this before as I wasn't the original source of the "register int" lines at ~124

C:\Users\useubuntu\CLionProjects\QUENOS\qcore.c:125:15: error: invalid register name for 'otherpidRegister'
  register int otherpidRegister asm("r22"); //read other pid for unblock
               ^~~~~~~~~~~~~~~~
C:\Users\useubuntu\CLionProjects\QUENOS\qcore.c:126:15: error: invalid register name for 'requestTypeRegister'
  register int requestTypeRegister asm("r23"); //read the type of request to determine what type of request is being made
               ^~~~~~~~~~~~~~~~~~~